digitalocean/kubernetes/kubernetes.go

```diff
@@ -240,6 +240,19 @@ func expandMaintPolicyOpts(config []interface{}) (*godo.KubernetesMaintenancePol
 	return maintPolicy, nil
 }
 
+func expandControlPlaneFirewallOpts(raw []interface{}) *godo.KubernetesControlPlaneFirewall {
+	if len(raw) == 0 || raw[0] == nil {
+		return &godo.KubernetesControlPlaneFirewall{}
+	}
+	controlPlaneFirewallConfig := raw[0].(map[string]interface{})
+
+	controlPlaneFirewall := &godo.KubernetesControlPlaneFirewall{
+		Enabled:          controlPlaneFirewallConfig["enabled"].(*bool),
+		AllowedAddresses: controlPlaneFirewallConfig["allowed_addresses"].([]string),
+	}
+	return controlPlaneFirewall
+}
```

Comment:
Awesome work, Fabian :rocket: :rocket: :rocket: :rocket: :rocket: 

```suggestion
		ExpandAllowedAddresses(controlPlaneFirewallConfig["allowed_addresses"].([]interface{})),
	}
	return controlPlaneFirewall
}

func ExpandAllowedAddresses(addrs []interface{}) []string {
	var expandedAddrs []string
	for _, item := range addrs {
		if str, ok := item.(string); ok {
			expandedAddrs = append(expandedAddrs, str)
		}
	}
	return expandedAddrs
}
```